### DISCUSSION TITLE
LNReader detected as infected app on ColorOS13

### DISCUSSION THREAD

**Discussion by kuadhual** [lnreader/lnreader#742] [Q&A]
Hi, I'm using an Oppo Phone (ColorOS 13). This is the second time I have to uninstall LNReader because it was detected as infected app by the phone. 

I upload LNReader-v1.1.17.apk on virustotal an it was detected as PUP/Android.Malct.518314 by AhnLab-V3 and APK:RepMalware [Trj] by Avast-Mobile. You can see the result at https://www.virustotal.com/gui/file/afc18dad898b4d62fdfc530a9d997ea25ceaad97a7cf83cf1e35fdfaa8e18b1b 

Even if it's  a false positive, the pop up that prompt me to remove the app is quite annoying.

**Comment by rajarsheechatterjee** [comment]
Duplicate of https://github.com/LNReader/lnreader/discussions/724
